app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split(" ")
+    first_command = command_list[0]
+    if first_command == "mv" and len(command) >= 6:
```

Comment:
This check `len(command) >= 6` looks useless, need to check if command contains 3 parts